server/src/main/java/org/elasticsearch/search/aggregations/bucket/composite/DateHistogramValuesSourceBuilder.java

```diff
@@ -80,12 +91,18 @@ protected DateHistogramValuesSourceBuilder(StreamInput in) throws IOException {
         super(in);
         dateHistogramInterval = new DateIntervalWrapper(in);
         timeZone = in.readOptionalZoneId();
+        if (in.getVersion().after(Version.V_8_0_0)) {
```

Comment:
I *think* this is the way to do this, but it has been a long time. I want to land this change in master and 7.x, but these tests have no change of passing the bwc tests until I backport it.

Now that I think about it, maybe *this* version check should be `V_7_6_0` actually. Help!

Reply:
The tests caught an error - this at least be `onOrAfter` but I'll switch it to `V_7_6_0` while I'm there.